vault-core/src/test/java/org/apache/jackrabbit/vault/packaging/integration/PrincipalBasedIT.java

```diff
@@ -201,6 +206,24 @@ private void assertPolicy(@NotNull Principal principal, @NotNull AccessControlEn
         fail("expected PrincipalAccessControlList for principal " + principal.getName());
     }
 
+    @Test
+    public void testEntriesFromSetup() throws AccessDeniedException, AccessControlException, UnsupportedRepositoryOperationException, RepositoryException {
+        @NotNull
+        JackrabbitAccessControlPolicy[] applicablePolicies = acMgr.getApplicablePolicies(testUser.getPrincipal());
+        // for some reason only 1 is returned and no longer 2 as in the setUp method
+        assertEquals(1, applicablePolicies.length);
```

Comment:
@anchela Do you know why evaluating principal ACLs set in [line 110](https://github.com/apache/jackrabbit-filevault/pull/293/files#diff-038077103e482e4a69670d008bbd1c150aab6edb9c6cb2d8e531940b094170e9R110) doesn't work?

Reply:
Nevermind, was a misunderstanding of `getApplicablePolicies` vs `getPolicies`. I will try to improve the javadocs in that regard in https://github.com/apache/jackrabbit-oak/blob/d25263450748fde7fd72923b5253cf637794886c/oak-jackrabbit-api/src/main/java/org/apache/jackrabbit/api/security/JackrabbitAccessControlManager.java#L48.